views/index.handlebars

```diff
@@ -199,6 +201,77 @@
 		</div>
 	</section><!--/#explore-->
 
+	<section id="about">
+		<div class="container">
+			<h2>How it works</h2>
+			<div class="row">
+				<div class="col-md-3">
+					<div class="about-step">
+						<img class="img-responsive" src="images/how-it-works/step1.png" alt="Download the app">
+						<h4>Install the app</h4>
+						<p>Get started by installing the app and help us spread the word.</p>
+					</div>
+				</div>
+				<div class="col-md-3">
+					<div class="about-step">
+						<img class="img-responsive" src="images/how-it-works/step2.png" alt="Download the app">
+						<h4>Form a team</h4>
+						<p>Playing with friends is a lot more fun! But don't worry, you can still play solo if you prefer.</p>
+					</div>
+				</div>
+				<div class="col-md-3">
+					<div class="about-step">
+						<img class="img-responsive" src="images/how-it-works/step3.png" alt="Download the app">
+						<h4>Pick a challenge</h4>
+						<p>There will be different challenges for different levels and categories to fit a range of needs and preferences. Everyone is welcome!</p>
+					</div>
+				</div>
+				<div class="col-md-3">
+					<div class="about-step">
+						<img class="img-responsive" src="images/how-it-works/step4.png" alt="Download the app">
+						<h4>Play!</h4>
+						<p>Start riding and have fun! When you bike, Emission will make sure your well deserved points are computed automatically. No data input hassle!</p>
+					</div>
+				</div>
+			</div>
+		</div>
+	</section><!--/#about-->
+
+    <section id="challenges">
+        <div class="container">
+            <div>
+				<div class="col-md-3">
+					<div class="about-step">
+						<img class="img-responsive" height=150 width=150 src="images/challenges/shankari/shankshallenge3.png" alt="Download the app">
+						<h4>Shankari: &#x025B;-optimization</h4>
+						<p>Can you get your travel to be &#x025B;-optimal? Short trips (&lt; 5km) are eminently walkable and bikeable, truly zero emission, and, around campus, can be faster than driving. The reward function for this challenge discourages deviations from this optimum, and provides a simple linear benefit for longer trips.</p>
+					</div>
+				</div>
+				<div class="col-md-3">
+					<div class="about-step">
+						<img class="img-responsive" height=150 width=150 src="images/challenges/kris/couch_potato.png" alt="Download the app">
+						<h4>Kris: Baby Steps</h4>
+						<p>Can you get your travel to be &#x025B;-optimal? Short trips (&lt; 5km) are eminently walkable and bikeable, truly zero emission, and, around campus, can be faster than driving. The reward function for this challenge discourages deviations from this optimum, and provides a simple linear benefit for longer trips.</p>
```

Comment:
TODO: This is currently copy pasted for everybody except me. I need to replace the summaries with real text. Think of this as lorem ipsum.